[#10988] Add entry point to start new chat from search results

Ferossgp opened the pull request:
Fixes #10988

Ferossgp committed:
[#10988] Add entry point to start new chat from search results

Fixes #10988

Allow public key and filter public chats

Make toolbar width 100

Clear new identity from search

QA issues fix

Signed-off-by: Gheorghe Pinzaru <feross95@gmail.com>

status-im-auto commented:
### Jenkins Builds
| :grey_question: | Commit | :hash: | Finished (UTC) | Duration | Platform | Result |
|-|-|-|-|-|-|-|
| :heavy_check_mark: | 68c56939 | [#1](https://ci.status.im/job/status-react/job/prs/job/android-e2e/job/PR-10997/1/) | 2020-07-27 08:02:43 | ~10 min | `android-e2e` | [:package:`apk`](https://status-im-prs.ams3.digitaloceanspaces.com/StatusIm-200727-075238-68c569-pr-x86.apk) [:calling:](https://chart.apis.google.com/chart?cht=qr&chs=400x400&chld=L%7C%0A1&chl=https%3A%2F%2Fstatus-im-prs.ams3.digitaloceanspaces.com%2FStatusIm-200727-075238-68c569-pr-x86.apk)|

flexsurfer commented on src/status_im/ui/screens/home/views.cljs:
```diff
@@ -96,7 +98,29 @@
+
+(defn start-suggestion [search-value]
+  (let [{:keys [state ens-name public-key]} @(re-frame/subscribe [:contacts/new-identity])]
+    [react/view {}
```
**just** wondering, why an empty map is needed here?

Ferossgp replied:
Yes, it is not needed here

flexsurfer commented on src/status_im/ui/screens/home/views.cljs:
```diff
@@ -96,7 +98,29 @@
-                       (re-frame/dispatch [:search/home-filter-changed text]))}]])
+                       (re-frame/dispatch [:search/home-filter-changed text])
+                       (when-not (seq chats)
+                         (debounce/debounce-and-dispatch [:new-chat/set-new-identity text] 300)))}]])
```
do we clean it later? for example, if a user presses cancel? i mean **just** to make sure if we open create new chat later input field will be empty

Ferossgp replied:
It is cleared when we open `start a new chat` screen. But I will add now also clear here, so if we will use in other palces too, it will be clean.

status-im-auto commented:
## 99% of end-end tests have passed
```
Total executed tests: 98
Failed tests: 1
Passed tests: 97
```
<h3>Failed tests (1)</h3><details><summary>Click to expand</summary><br/><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1573860">test_fetching_balance_after_offline</a></b></td></tr><tr><td><p><blockquote><div>Device 1: Wait for EnterPasswordInput</div><div>Device 1: EnterPasswordInput element not found</div></blockquote></p><code> Device 1: 'EnterPasswordInput' is not found on the screen</code><br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/14983a4154f54fd28e67b792a9a68c03?auth=034fa8720604da64713a0070f4fc9920">Steps, video, logs</a></li><li><a href="https://assets.saucelabs.com/jobs/14983a4154f54fd28e67b792a9a68c03/0040screenshot.png?auth=034fa8720604da64713a0070f4fc9920">Failure screenshot</a></li></ul></p></ul></p></td></tr></tbody></table></details><h3>Passed tests (97)</h3><details><summary>Click to expand</summary><br/><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1573869">test_decline_transactions_in_1_1_chat</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/0893968e73234f3da7ae0fe9ef3b327f?auth=1e789083a6cce2638f2dba55a36ee002">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/eb705969a6ee4aa2813849f97bf62d2c?auth=51ac64698c62fe581d9b8ba4278e4c6c">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1573815">test_delete_chats_via_delete_button</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/2b45e4d52ceb4306a7df79cd62249957?auth=fcc8b7f1d60810b0a965aba7ca045301">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1573775">test_can_see_all_transactions_in_history</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/28a0a67acbe243dcb94fdfcf514072c7?auth=4e885bd202cc239780de151fca73d75c">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1573845">test_install_pack_and_send_sticker</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/686f82813f42458981c558de2648f0e2?auth=bc5e03cbfa060ff40385580326a120d6">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/9e06a671ab584cf594e9a6f2dbc00a95?auth=196230aab2dfa221f9c44c4ebf93e2eb">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1573773">test_login_with_new_account</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/b31f74073c11466f9f90a53e5923ccda?auth=1f43e060fcc4dca1dfbdc2963e061656">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>6. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1573813">test_timestamp_in_chats</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/508dda04dc954f1092f91ef8209c77d0?auth=1403fb108fc346472a5f3e1602c0861e">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/2a0f0713863a4b669cb87c33ba477515?auth=b82d446ca7719beb951f196a4a6024b0">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>7. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1573828">test_password_in_logcat_sign_in</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/231155c4bdab4b4793e9dc52cb49bef7?auth=fa8b987a8f21d2055f63e18c3cd6b066">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>8. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1573805">test_logcat_recovering_account</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/4c68a8699039485bb44a1ce0c5590f51?auth=ad2fcfc511a0819b7513990df48af8a7">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>9. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1573814">test_public_chat_clear_history</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/65bd24ee774a48db9adc8b67713d7100?auth=c56f0a09e5c78dbb53a4e163d18a4b19">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/95b8b0396cc64cbdbbb6c59d616f5fb1?auth=b2bd3e7039b1dfb8014b386b3f73eae8">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>10. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1573817">test_need_help_section</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/b443f9bfc5ba49ddb26242773e5ca033?auth=16fe58cc4396198002c5bc7d2845e47e">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>11. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1573807">test_send_and_open_links</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/522022f3b6ea4c32a7837e03c92f0833?auth=143f2e7ad10b5201b494104d803e4264">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/5e2d1b17c52a40dcbf8274610f901d2b?auth=e75ea89cd0bba31f66d08a43a209461c">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>12. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1573858">test_mobile_data_usage_popup_stop_syncing</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/a098415fc6184a7f81c22784c69685f6?auth=83ae90630f8866f738b79905031526ad">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>13. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1573882">test_keycard_send_eth_in_1_1_chat</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/ab65627419994fd39e7c48c5e9231265?auth=d03a060267e3139e592df914b1842419">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/bbf8fa9ba5c1415184c325029a15d386?auth=3d151dd2fb8cd13f351e275c8ff8c8e1">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>14. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1573859">test_mobile_data_usage_settings</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/b2ece49a2ca24ec2a663ce56ac1e227d?auth=de4646034c239574c0db9a7100e030c5">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>15. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1573764">test_create_new_group_chat</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/0a978eb79b29497a878ce59fbf01a029?auth=710cc24f20078a620607ddf0b23d1f0c">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/b2391f13635f4fb3ba6b223636ef39f2?auth=f17d095d93f73530ac9a159bb79d5a5c">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>16. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1573884">test_keycard_send_eth_to_ens</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/d69cb2d8d84442ba88a93afa376db6a0?auth=e6b0d7b3af2900c1d572f55ab2233d82">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>17. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1573766">test_user_can_switch_network</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/2db0e419d42d4581910e4f0a85220406?auth=1813af17f90d3f9043a85b83467bdf00">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>18. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1573838">test_mobile_data_usage_popup_continue_syncing</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/d6467012891c4863a7c772bac687f4a7?auth=becb15e75830211e3391a6b36d83b071">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>19. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1573809">test_home_view</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/ac812167a4534d2185b35903d9517206?auth=58de9cf8563b9c8f1196ff6fcc801a75">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>20. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1573780">test_open_google_com_via_open_dapp</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/c91545185a2a42848164cf2db676b1e0?auth=aed31154f5b5af656e25697575b6907e">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>21. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1573782">test_share_contact_code_and_wallet_address</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/3cc8473202b14b1eb94ee9607e359a35?auth=02ac6d5f30f5d8e96141df05c9b63666">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>22. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1573822">test_connection_is_secure</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/65d9658ef2674028b3217badea630d09?auth=7a239d2222eefa93b768df01d9c896ce">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>23. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1573777">test_add_to_contacts</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/fb95ab349bfc43459180a1ed34a06266?auth=e654d5a153fc9fa105fd154d3244499b">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/e1720c2ea2244d4a9f896d80e756ec15?auth=478d5ecef0ba94e88732ea65d164af34">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>24. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1573871">test_add_account_to_multiaccount_instance_private_key</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/d10c3bed6c32459388fb0361ecc7d401?auth=47335a4c2091c1fe77a7e1de8c7a9dce">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>25. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1573797">test_refresh_button_browsing_app_webview</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/cae38c577ba24d4ba7740b82638e72d7?auth=c2b39ae5e3287b36e76db17e2c280e1f">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>26. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1573875">test_push_notification_1_1_chat</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/7e8ba02aaf9b4958986bf7dcab3494cf?auth=ee2be8f1469c535c31f0833e49e1617e">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/6c2f48bca1de4c4fbd43ebf0a088f448?auth=80e858811d7b8abe39e710c062199077">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>27. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1573878">test_keycard_send_two_transactions_one_after_another_in_dapp</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/7849e0252fab4e5a902b36e8fd73a83b?auth=05c79a5ffdf1d431d3ec1831639d982d">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>28. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1573831">test_can_add_existing_ens</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/5ca1fa5c72e246b684f49eaf0be46bdc?auth=d58b762aab4f9178b1cf98575a6ba176">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>29. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1573825">test_transaction_wrong_password_wallet</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/e963ee7a65d74758b14dde6cbd6293c0?auth=063db6c79839b3248167a61888b3c1e3">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>30. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1573774">test_public_chat_messaging</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/197bdf5a36854371b2a85fd53ddbdad8?auth=44b2e624fd8b4a4c53a7bd9f2572129a">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/e05f25a855f549ccb1d5f736ae144cc2?auth=ecec01f5c9bd33983b8a8580bee2145d">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>31. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1573800">test_unread_messages_counter_public_chat</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/49078fc6b4054c5b8121b6bf5a5026f1?auth=7dfc34a68da1cf7d43bc8ce8f9cc47b1">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/a154b6c4053747428526d7595a565df4?auth=30599cf09618666bb28f0b25e9c31fde">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>32. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1573879">test_keycard_send_eth_from_wallet_to_address</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/67093b8e7f6f4372be705bba4eb3e91f?auth=9733c46b8796b8825fcec4b017e3d2d5">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>33. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1573880">test_keycard_fetching_balance_after_offline</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/6cd17d30eff04d2fa84e5a476c9b85e2?auth=9d75a2b4ce6d35823cc75428e40dfaf2">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>34. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1573863">test_keycard_send_transaction_from_daap</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/3c045795a9694e17b2cb58e19088408f?auth=2a17e77b18247feff41b402c3d303a25">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>35. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1573769">test_text_message_1_1_chat</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/404f715d7a0b47c39d39456709610ee0?auth=c6dd0d418c3351d8a61815104ec788da">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/0c351d31e80948c19c118b199867e986?auth=29b996f9b690371fb215c0136824a68f">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>36. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1573846">test_can_use_purchased_stickers_on_recovered_account</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/a1214951fe1f4bb68a139330d28099b2?auth=e9c27cc3a5970f34a123e6e1e27f6538">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>37. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1573812">test_open_transaction_on_etherscan</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/4e5e35ea5ca94660ac57fac4fce3f0ee?auth=4a4ad597225d38e20fe039afe6f9cb1d">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>38. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1573834">test_onboarding_screen_when_requesting_tokens_for_recovered_account</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/5e95231f2ce64566b638604a5c9b26cf?auth=45c13d36543757a77b3e5124df40b132">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>39. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1573811">test_contact_profile_view</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/2c31dc077ce842b9a2df0d650551929c?auth=09586284845511920fb1153b76ebc5ed">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>40. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1573765">test_offline_add_new_group_chat_member</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/cefe1415a86f4857a6840ba728fa3c85?auth=f851bc2317db686ea8ea0de75cc4e734">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/17c469ffc7994cd2a36db8f2991bfd1c?auth=4781c1e0ae99cba4166371632700ef1f">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://saucelabs.com/jobs/f2cecc0867904b0db60d7dac87139279?auth=441abbbb61f380493736aba052855d81">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>41. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1573829">test_logcat_backup_recovery_phrase</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/594d1cebb7af42fd804b6960a2450c67?auth=955fe1f91481112d66026f24de90b720">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>42. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1573841">test_can_recover_keycard_account_card_pairing</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/0e75c9f3323745b28b64ad484f9510e0?auth=398f6f61918163964accc8a2792e3a6c">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>43. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1573823">test_start_chat_with_ens</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/0f5901ef23b4495390d4c5e3377eddfc?auth=477388f5c443cec67a74f920f6941857">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>44. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1573786">test_add_and_remove_contact_from_public_chat</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/ca458bbcc13440038a8e2d3b29a8294b?auth=93b0d6fb2b6148e8c3274e011fb47dd3">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/8d81f8da5a3845fa8280c9c0849509c8?auth=933958b45d8c66a280a6f5a0291337a3">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>45. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1573870">test_add_account_to_multiaccount_instance_seed_phrase</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/6e9d1e0b516e4e828a21168bc1c9b0e8?auth=609cb6d9db4b160f6e3bbb41748a99fa">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>46. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1573851">test_open_blocked_site</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/70530ea003b348dfa42f1e61bf79cd2c?auth=7c9123d2185bc24554b97ac4852ad42e">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>47. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1573768">test_open_chat_by_pasting_public_key</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/10c96fb096fb4a5ea27c38f04212e8c0?auth=c710220e68f49417c8ce516b1c1c68de">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>48. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1573843">test_pair_devices_sync_one_to_one_contacts_public_chat</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/969fab4955c24d9cb3a9841fb66ae6a3?auth=44cb5613b141e29a246d63b0b42626ff">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/8e25b1d85fc04065a1df1047c508d1b9?auth=29cae30ce7cf2395e15876255e54a7b2">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>49. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1573865">test_keycard_sign_message_from_daap</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/9846532f2f2a4107903eb477c60bf19e?auth=7dac9adc3b1d494879a09c33c9e90a8f">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>50. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1573776">test_send_non_english_message_to_newly_added_contact</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/53444d9758914849a71fb225a51808e4?auth=8141b1109ffc0984e6e979c44908485f">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/6b634b710754475ea60b023f72255dd8?auth=ea582f1c971ac0322408d3f5aa4d3716">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>51. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1573824">test_ens_username_recipient</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/985386b8da57464eada3a45adf2105b2?auth=fe1caa4778fdd9b9aa0c7bb1a3c7010a">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>52. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1573848">test_fetch_more_history_in_empty_chat</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/58ec5bbae89e45b098242374c4921ab8?auth=590df74409c15a6aa25fc7e9c4d9c7ac">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>53. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1573820">test_open_public_chat_using_deep_link</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/af29fcced2d14ca0b16506c26f2ff41e?auth=5efae4d62d77456361bb06ecda7d12d5">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>54. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1573798">test_switch_users_and_add_new_account</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/0ce87ec7f51e457b9502a6ed4addedee?auth=009c5f9a42ca439802eae4ff4a913986">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>55. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1573855">test_add_account_to_multiaccount_instance_generate_new</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/21de96014d5343e68c041b1ccab4f65d?auth=78f49b1bf05db93d5927d28e68d9e3fe">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>56. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1573808">test_copy_contact_code_and_wallet_address</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/1bab1ad422504afc9d2b1bc7275871a0?auth=0f111a4afedf6debd272abe45234db57">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>57. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1573771">test_send_transaction_from_daap</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/31409319ddad435d8dbc2742e11d0065?auth=c50f62a7fe8ee4ad0ebad6e8481a318e">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>58. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1573840">test_send_two_transactions_one_after_another_in_dapp</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/b592de24f1744b38ad0ab9b96263e2dc?auth=4b9b022c9b7c016ebc4de5d0234705be">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>59. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1573886">test_image_in_one_to_one_send_save_reply</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/5b0376d8e0a747ef9b3f1e04f1b2bb1e?auth=3423b742b0b322fbbc98db49cc1a3cc8">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/aabf233fa8b6431680f1d0a852660c64?auth=a16fa756bfd5043eae84d56c35cdf02e">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>60. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1573835">test_add_new_keycard_account_and_login</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/d6fb27fc03c64da1bc17f03aa456408a?auth=c9dfc1a380224b93db4b8f3f631ce7bb">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>61. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1573816">test_long_press_delete_clear_all_dapps</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/1ef9e9dc71404fbcbecadfcbbb79e192?auth=f8d5eae01961f0f3102fe6fcf1cacff6">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>62. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1573778">test_copy_and_paste_messages</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/e626013d01634b32982fcd17d2c8e92e?auth=51d211daefb704c7e651d84414041a47">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>63. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1573781">test_back_forward_buttons_browsing_website</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/1d15ddae527c482394fac9b20ec42534?auth=9d9af69e96e0486f0bf85f1223773ee8">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>64. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1573827">test_password_in_logcat_creating_account</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/aefbcff3ea4d45b293a5fd59ea7df20b?auth=53fbb0645ed7a1cd0b45027174ecd69b">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>65. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1573793">test_sign_message_from_daap</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/940acefa712c46f6a8a00d6155d06f75?auth=a71256e39d4e7570c893f523f52e191f">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>66. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1573837">test_dapps_permissions</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/bfa7e7fed7e5417f8208e4130c04e42d?auth=7b0d2fd7550409a73ef8788deebea75a">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>67. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1573821">test_request_public_key_status_test_daap</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/015e6e9fcc9f48ab969927fca44b207c?auth=e0e063d8d8edf872afcf76614c7f642a">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>68. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1573779">test_long_press_to_delete_chat</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/bba8f58f2bcb4e689971875918e187d2?auth=d271b68338d790e9a244e7406672a212">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>69. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1573806">test_log_level_and_fleet</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/22759291622c4b65bb4e3fd28ac24adc?auth=f6e95f57680edb1a2bc14679392ea133">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>70. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1573862">test_add_and_delete_watch_only_account_to_multiaccount_instance</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/884a76e472384f98a42c3ce482275d6e?auth=8cd6cd8f750feaa2d4bb5d8414d94f2b">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>71. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1573885">test_recover_account_from_new_user_seedphrase</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/81bd18c64c694b078b9ded5a0dbefe8f?auth=deff8d9a0ca87bdaf7e49d2331d35809">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>72. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1573849">test_send_transaction_with_custom_token</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/62c4e83f5f3d4e908005c80496dd7bea?auth=a51fc0360b1b0bec5e60f494123ed5d1">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>73. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1573883">test_keycard_request_and_receive_stt_in_1_1_chat_offline</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/9b889604da7e4fa998b6d685b1f9ce6a?auth=61eacc2d7365288babbc6913f6d2804d">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/7abefcf9af604e5fb045223f94d47704?auth=38ae1cb15e015f4c3de6816b167c7c21">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>74. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1573866">test_send_eth_in_1_1_chat</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/d5ea88a8822d435ab6d014382b21db4f?auth=c79e2205e3e2b6e6b67a58184fef9b12">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/0a4a76f8295544c58e01135eb5fce943?auth=f4ca8d3b4db95dae341b25f096297e10">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>75. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1573839">test_send_two_transactions_in_batch_in_dapp</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/601ea01f94a34552be1b75c46773340e?auth=01419727738f7b15b73d6f786e0599ce">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>76. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1573856">test_ens_in_public_and_1_1_chats</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/0bc002160fff49938cba1d5c2e3273b2?auth=ca644fa32f82e1cac9c1e1f2d21b210d">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/45d6c9932b7b4efb838cd6a98ee5df68?auth=908d59d6de201bb0585205ebdfa84d2f">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>77. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1573847">test_block_user_from_public_chat</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/7499ba1b18ce4fd68c5a7cb1401d13af?auth=25205fe20a50babfa05b901e6d76df73">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/9c67d473f4c34b33a6e3310239c49b98?auth=8a0e89027119ec20fc60390731ef1da5">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>78. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1573783">test_offline_status</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/b93f9f73dfd84df8968b612a3d74b097?auth=46603997e20f18822e46e27ab3c07353">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>79. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1573787">test_wallet_set_up</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/4befb4100d9b47979a4656d77d5b7c4b?auth=d933a0aa3efd2866b33c68ec067d45b3">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>80. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1573792">test_manage_assets</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/429faf76f772471c81d7ec7e88e2e91d?auth=c377aef71b11a0b87ef8e26710a6157f">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>81. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1573772">test_offline_messaging_1_1_chat</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/89ef8e4a31ec45329e427ed921ed23fe?auth=77bf6a6204a7aa2c3ace48ea1d5f9210">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/d9b510a45ef34cc3a9eb06f8b2b29d0f?auth=8af578893b9ee3b37ec57cb2f9b7ee54">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>82. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1573818">test_account_recovery_with_uppercase_recovery_phrase</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/616601ac52994490bcd76bac43e282eb?auth=21e34ba8048d8efa4384429dee14c028">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>83. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1573795">test_send_token_with_7_decimals</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/327176677297475ab63d4860a6d2c2ee?auth=527ddde7a8a2deb7160c45ddd325fc8a">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>84. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1573810">test_user_can_see_all_own_assets_after_account_recovering</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/6cb2d3325b17446dbadf30a093b621bc?auth=626814ac83055b92b2535c01124480bb">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>85. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1573802">test_pass_phrase_validation</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/59ef2baf340f42b7a2a0656b8f6b509a?auth=3dc796ed96d234f73f5ee548e452ac0c">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>86. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1573881">test_keycard_can_see_all_transactions_in_history</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/082a40ea5623477a8faf12eea9b7e3f2?auth=2258930bc5a67ccd0a3fc7272e84d91d">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>87. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1573877">test_keycard_send_two_transactions_in_batch_in_dapp</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/2f83493f19204afbb4c4b9a93991712f?auth=1ceab6338f8930e97d618c805d9b1d52">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>88. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1573784">test_send_emoji</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/b542e4a5bafa42cebe772174da55b1aa?auth=ad0d905fb2b0a3c01448e9ab6678bd68">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>89. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1573796">test_filters_from_daap</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/a74a20b5aa1642588bf51cd57defe5d9?auth=15d1168cdc1d67c627a4432ef7e662ac">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>90. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1573833">test_redirect_to_public_chat_tapping_tag_message</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/bd5cdd8cfa5b444a844edb76fd5068be?auth=9ed93ae34e9955271bc8aef4afed5e07">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>91. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1573861">test_restore_account_from_mnemonic_to_keycard</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/517849fcf82548f1b55889e962acaf29?auth=42fae11bdddd44f4a0abc118dcefbed3">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>92. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1573873">test_send_eth_to_ens_in_chat</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/59f6cda6fe3449109b4b699c17f359ff?auth=24ad35f39b94e281df1b490526360baa">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>93. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1573770">test_send_eth_from_wallet_to_address</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/a584c59e18e24887ae331f381806ed3f?auth=a517defd1e4c53870229aee66c8a7539">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>94. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1573868">test_request_and_receive_stt_in_1_1_chat_offline</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/b6d1af6c6ffb4ec3a5c79b5d0c57e2cb?auth=d7092374c5539882574ad829e727d2bd">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/8ed067d3afbb4caf99989ef3ddfb849c?auth=cc98442bf7fd2b402a91f587266e6344">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>95. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1573789">test_messaging_in_different_networks</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/bfb33f90c90f4e0692dcd566eaabd956?auth=35684fb30de4bf6dddf85f81fcc1f70f">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://saucelabs.com/jobs/0eebce1a1aa64782872416f44b443a3e?auth=7630f91cf72d06025f5b71d06c0b580a">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>96. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1573794">test_collectible_from_wallet_opens_in_browser_view</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/8a97f28e5c3e49eaa6d3d5df248c3c14?auth=b2a5f953fe73f514032811bdad28f546">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>97. <a href="https://ethstatus.testrail.net/index.php?/tests/view/1573826">test_insufficient_funds_wallet_positive_balance</a></b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://saucelabs.com/jobs/04ba404205e144558c3aea2cfbc0c652?auth=ae051c352e040ddade77b4e3b7fb34e7">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table></details>

churik commented:
@Ferossgp 
### ISSUE 1:  Error when tap on public chat name under "Do you mean"
*Steps:*
1) join any public chat
2) type 'aaaaa' in search
3) tap on 'aaaa' under Do you mean

**Expected result:** redirect to public chat
**Actual result:** 
![IMAGE 2020-07-27 18:15:18](https://user-images.githubusercontent.com/4557972/88565601-20c75580-d035-11ea-8d25-d89330475f48.jpg)

### ISSUE 2:  No chats are suggested if I type existing ENS with domain 

I.e. when typing 'tanyatest1.eth' or 'qbnewzz.stateofus.eth' I would expect that 1-1 chat is suggested

### ISSUE 3: Search input is not cleared after redirecting to chat

*Steps:*
1) join any public chat
2) type 'tanyatest10' in search
3) tap to start 1-1 chat
4) tap back

**Expected result:** redirect to home view, search input is cleared
**Actual result:** redirect to home view, in the search field the same value is set

### ISSUE 4: 1-1 chat is not suggested in case of pasting ENS to search input

**Steps:**
Scenario 1: 
1) join any public chat
2) paste 'tanyatest10' in search

**Expected result:** 1-1 chat will be suggested
**Actual result:**  nothing happened

Ferossgp commented:
@churik 

> ISSUE 1
Fixed
> ISSUE 2:
Most probably the connection was slow? We do not show spinner for loading but use the same logic as start a new chat. I've tried these ens and it worked.
> ISSUE 3: Search input is not cleared after redirecting to chat
This is the same behavior as searching other chats
> ISSUE 4
Fixed

qoqobolo commented:
@Ferossgp 

### ISSUE 5: Search input disappears after deleting public chat

_Steps:_

1. Join any public chat
2. Type random word in search
3. Tap "Join a public chat"
4. In opened chat tap 3 dots on the top right 
5. Delete chat -> Delete

**Expected result:** redirect to home screen, chat list and search input are shown
**Actual result:** redirect to home screen, no chat list and search input 

Video: https://drive.google.com/file/d/1-sJDkNWG5bQnBmX6bBdNavR87_cQBdNP/view?usp=sharing

Ferossgp commented:
@qoqobolo Because this is also present in current release and. It a regression of this PR, I would prefer to have it as a separate issue to be solved independently of this task

qoqobolo commented:
@Ferossgp Yes, you are right about both issues (3 and 5)! Thanks for a great job and sorry for the distraction.

qoqobolo commented:
Tested latest build which are https://status-im-prs.ams3.digitaloceanspaces.com/StatusIm-200728-101749-cb4b02-pr-universal.apk and https://i.diawi.com/ox3cbt on Android 8, Android 10 and iOS 13.

**Scenarios verified:**

1) Join 1-1 chats and public chats and search for them (typing value, pasting value)
2) Typing and pasting chat’s name that are not in my chat list and joining them via “Do you mean” links: 
- ens names (no domen,  with .stateofus.eth, with .eth)
- valid values for public chat names, but not valid ENS
3) Invalid values + adding chars in the end of value / deleting chars in the end
4) Navigating back and forth after some actions in chat view (deleting chats, clearing history)
